src/asicverifier/src/main/java/ee/ria/xroad/asicverifier/AsicVerifierMain.java

```diff
@@ -139,6 +142,11 @@ private static void writeToFile(String fileName, String contents) throws IOExcep
 
     private static void showUsage() {
         System.out.println("Usage: AsicVerifier "
-                + "<configuration path> <asic container>");
+                + "<configuration path> <asic container>"
+                + "\nor: java -jar asicverifier.jar version");
+    }
```

Comment:
maybe simply: `Usage: java -jar asicverifier.jar [--version] <configuration path> <asic container>`
Usage could also reproduce the version information.